libs/src/api/track.js

```diff
@@ -341,12 +341,27 @@ class Track extends Base {
         metadataFileUUID,
         transcodedTrackUUID,
         transcodedTrackCID
-      } = await this.creatorNode.uploadTrackContent(
-        trackFile,
-        coverArtFile,
-        metadata,
-        onProgress
-      )
+      } = await retry(async (bail, num) => {
```

Comment:
since uploadTrackContentToCreatorNode has the retry i think we can keep this here

Reply:
yeah i think we want this one. we didn't have it before